src/BinaryParsers/PortableExecutable/PE.cs

```diff
@@ -522,6 +523,35 @@ public bool IsXBox
             }
         }
 
+        public bool IsBoot
+        {
+            get
+            {
+                if(null != _isBoot)
```

Comment:
Style nit, C# doesn't permit unexpected assignment in conditionals, so it isn't typical to order args for a null check in this way. if (_isBoot != null) would be more typical